subtree-splits/blt-project/composer.json

```diff
@@ -0,0 +1,82 @@
+{
+    "name": "acquia/blt-project",
+    "description": "Project template for Drupal 8 sites built with BLT.",
+    "type": "project",
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "license": "GPL-2.0-only",
+    "repositories": {
+        "drupal": {
+            "type": "composer",
+            "url": "https://packages.drupal.org/8"
+        },
+        "asset-packagist": {
+            "type": "composer",
+            "url": "https://asset-packagist.org"
+        }
+    },
+    "autoload": {
+        "psr-4": {
+            "Acquia\\Blt\\Custom\\": "blt/src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Drupal\\Tests\\PHPUnit\\": "tests/phpunit/src/"
+        }
+    },
+    "config": {
+        "sort-packages": true
+    },
+    "require": {
+        "acquia/blt": "dev-master",
```

Comment:
This needs to be updated to something like ^10.0.0 in coordination with the release of this PR.